app/src/common/shared/org/mozilla/vrbrowser/ui/NavigationBarWidget.java

```diff
@@ -529,20 +460,14 @@ public void onCloseRequest(GeckoSession session) {
     public void onFullScreen(GeckoSession session, boolean aFullScreen) {
```

Comment:
very nice work here! Fullscreen appears to be handled correctly with the Resize Mode changes :+1:

it worked as expected with YouTube videos I tried. I just hope it's obvious to users that they have to press the analogue back button on the VR controller (or if the page calls `document.exitFullscreen()`).